src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/media/VideoComponentObject.java

```diff
@@ -56,23 +56,23 @@ public void verifyVideoOoyalaEmbed() {
     String containerId = "ooyalaplayer-";
     Assertion.assertStringContains(container.getAttribute("id"), containerId);
     wait.forElementVisible(container.findElement(By.tagName("object")));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "object" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)